### DISCUSSION TITLE
Error  Wrong Password

### DISCUSSION THREAD

**Discussion by Vamps023** [masqu3rad3/tik_manager4#65] [Q&A] [answered]
Hey 
I try on Blender But not working or i am doing some wrong step Can Some one guide me to open and work on this 

I am getting Always Same error 
(Wrong Password provided for user admin)

![image](https://github.com/masqu3rad3/tik_manager4/assets/79307556/fc20650e-9421-461b-9bbc-dd1fc99459d3)
![image](https://github.com/masqu3rad3/tik_manager4/assets/79307556/0592a085-0c15-4dcf-a6ee-f7d41773d9b8)

I am using Same Password  but it still showing Same error

**Comment by masqu3rad3** [comment, marked as answer]
Hi,

That is not the password, thats just the hash of the password.

All default passwords for all users are 1234
There will be a user management dialog coming soon to organize these better
